spyder/plugins/ipythonconsole/widgets/main_widget.py

```diff
@@ -1519,16 +1519,23 @@ def create_new_client(self, give_focus=True, filename='', is_cython=False,
                 version=SPYDER_KERNELS_VERSION)
             if not has_spyder_kernels and not running_under_pytest():
                 client.show_kernel_error(
-                    _("Your Python environment or installation doesn't have "
-                      "the <tt>spyder-kernels</tt> module or the right "
-                      "version of it installed ({0}). "
+                    _("The Python environment or installation whose "
+                      "interpreter is located at"
+                      "<br><br>"
+                      "{0}<tt>{1}</tt>"
+                      "<br><br>"
+                      "doesn't have the <tt>spyder-kernels</tt> module or the "
+                      "right version of it installed ({2}). "
                       "Without this module is not possible for Spyder to "
                       "create a console for you.<br><br>"
-                      "You can install it by running in a system terminal:"
+                      "You can install it by activating your environment (if "
+                      "necessary) and then running in a system terminal:"
                       "<br><br>"
-                      "<tt>{1}</tt>"
+                      "{0}<tt>{3}</tt>"
                       "<br><br>or<br><br>"
-                      "<tt>{2}</tt>").format(
+                      "{0}<tt>{4}</tt>").format(
+                          '&nbsp;' * 8,
```

Comment:
Checking a little bit I think the same indent could be achived wrapping the respective lines with a `pre` tag and leaving four spaces. So something like this:

```python
"<pre>"
"    <tt>{1}</tt>"
"</pre>"
```
I'm not totally sure what version is better in terms of readability (`pre` vs `&nbsp`). Leaving the `pre` option here just in case.

What do you think is better @ccordoba12 ?

Reply:
Thanks for the suggestion @dalthviz! I recalled there was a tag to show code but I couldn't find it quickly, so I decided to to go for the ugly `nbsp` solution. I'll do the change.